eureka-client/src/main/java/com/netflix/discovery/shared/EurekaJerseyClient.java

```diff
@@ -218,6 +219,8 @@ public CustomApacheHttpClientConfig(String clientName, int maxConnectionsPerHost
             cm.setDefaultMaxPerRoute(maxConnectionsPerHost);
             cm.setMaxTotal(maxTotalConnections);
             getProperties().put(ApacheHttpClient4Config.PROPERTY_CONNECTION_MANAGER, cm);
+            getProperties().put(PROPERTY_FOLLOW_REDIRECTS, Boolean.FALSE);
```

Comment:
Can we add some comments on why the jerseyClient's redirects are explicitly turned off? Otherwise it may be confusing for future maintainers.